feature_tests/cpp2/include/AttrOpaque1.h

```diff
@@ -6,6 +6,10 @@
 #include <stddef.h>
 #include <stdbool.h>
 #include "diplomat_runtime.h"
+#include "AttrEnum.d.h"
+#include "AttrEnum.h"
+#include "Unnamespaced.d.h"
+#include "Unnamespaced.h"
 
 #include "AttrOpaque1.d.h"
 
```

Comment:
Nit: Maybe for another PR but we really seriously should get rid of the `namespace capi` before launching Cpp2. The C++ objects should always be in _some_ namespace and the C objects should sit in the global namespace with a prefixed name.

Reply:
https://github.com/rust-diplomat/diplomat/issues/432